blockly-core/core/ui/contract_editor/example_view.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+goog.provide('Blockly.ExampleView');
+
+/** @const */ var NO_RESULT_TEXT = "Test result: not ran yet.";
```

Comment:
i18n. Also, is this proper grammar? It doesn't sound quite right to me, though I don't say that with 100% confidence.